spec/diddocuments.md

```diff
@@ -89,19 +89,15 @@ The value of the `controller` property MUST be a single string that is the same
 ```
 
 #### Also Known As
-The `alsoKnownAs` property in the root of the DID document MAY contain other equivalent,
-resolvable `did:webs` DIDs. The `alsoKnownAs` property MAY contain `did:web` versions of the `did:webs` DID(s).
-
-It is anticipated that implementations of this DID method will be able to serve the same AID
-as multiple DIDs, all of which are synonymous for each other.  Any implementation will be able
-to provide the URL endpoint required to serve any AID it is serving as a `did:webs` DID as for
-`did:web` resolution.  Likewise, any implementation should be able to serve any AID it is serving
-as a `did:webs` DID and as a `did:keri` DID as well.  Finally, the same AID may be served under
-multiple domains at the same time and they should be considered the same DID since the AID portion
-of the DIDs are the same.
-
-For each synonymous DID defined above (TODO: we need a way in KERI to declare other domains it is being
-served under, unless this is an implementation specific detail) an entry in the `alsoKnownAs` array
+The `alsoKnownAs` property in the root of the DID document MAY contain any non-`did:webs` DID that has the same AID, see [[ref: AID controlled identifier]].
```

Comment:
What's the reason for not including `did:webs` DIDs in `alsoKnownAs`? Sorry, maybe we discussed this already and I forgot?

Reply:
> What's the reason for not including `did:webs` DIDs in `alsoKnownAs`? Sorry, maybe we discussed this already and I forgot?

We felt it is important to distinguish between the two. EquivalentId is the 'most restrictive' (same AID, same method). alsoKnownAs is the 'least restrictive' (same AID, not the same method, and we even discussed that it might not have to be a DID at all?).

Reply:
I have updated it now to include equivalentIds in the alsoKnownAs